src/commands/dev.ts

```diff
@@ -207,7 +207,14 @@ export default class Dev extends BaseCommand {
       compose_args.push('-d');
     }
 
-    await DockerComposeUtils.dockerCompose(compose_args, { stdio: 'inherit', env: { COMPOSE_IGNORE_ORPHANS: 'true' } });
+    const docker_compose_runnable = DockerComposeUtils.dockerCompose(compose_args, { stdio: 'inherit', env: { COMPOSE_IGNORE_ORPHANS: 'true' } });
+    try {
+      await DockerComposeUtils.watchContainersHealth(compose_file, project_name);
```

Comment:
Should this be an await? I got in a bad state where I couldn't ctrl-c because of the while loop (I think). Had to manually kill the processes.

Reply:
Nevermind that might be the kill call I added. Was having a separate issue where the docker-compose process never ended if watchContainersHealth threw an error.

Reply:
It actually does not have to be so will remove this.